### ISSUE TITLE
parse_json returns int and not the new dictionary

### ISSUE THREAD

**Issue by AlexHolly** [godotengine/godot#3011]
Two lines to parse a json that could also be done in one line.

See some Examples:

``` GDScript
# NOW
var dictionary = {}    
dictionary.parse_json(response)
return dictionary

# SUGGESTION
return {}.parse_json(response)

# OR
return Dictionary.parse_json(response)

# OR
return response.to_dict()

# OR
return response.parse_json()
```

**Bugsquad edit:** embedded code from http://pastebin.com/23yDJw36 to keep it permanently here.

**Comment by vnen** [comment]
As I said on IRC, I'm more fond of `var result = Dictionary.parse_json(string)`, i.e., `parse_json()` should be a static function of `Dictionary`.

**Comment by punto-** [comment]
Sometimes json strings are arrays and not dictionaries tho.

**Comment by reduz** [comment]
it's noted to be modified in the future (3.0) version but will not be
changed for now to keep compatibility

**Comment by bojidar-bg** [comment]
Pretty related to #4232 (and the solutions mentioned there)

**Comment by akien-mga** [comment]
Fixed in 62273e5.
